Embedded d2 diagram spawns Chrome, hangs, then terminates

## What happened?

Embedding a D2 diagram in a Markdown file, then running Mark on the file hangs until a timeout is exceeded. I can see a Chrome process running in the background, which apparently isn't doing anything.

## What did you expect to happen?

According to the documentation, a PNG image is supposed to be generated to be used as attachment. 

## How can we reproduce the behavior you experienced?

First, create this test file as test.md:

```
<!-- Space: X -->
<!-- Parent: Y -->
<!-- Title: Z -->
  
```d2
x -> y
``\` (Remove the backslash here, I have no idea how to escape the block for Github)
```

Now, run mark: `mark --compile-only --features d2 -f test.md`

After a while, the following output is produced and mark terminates:

```
2025-06-11 20:59:07.865 INFO  processing test.md
panic: context deadline exceeded

goroutine 1 [running]:
github.com/kovetskiy/mark/markdown.CompileMarkdown({0xc00003dd70, 0x11, 0x18}, 0xc00046ef40, {0xc000529379, 0x7}, {{0xfa05e2, 0xb}, 0x3ff0000000000000, 0x0, ...})
        /home/runner/work/mark/mark/markdown/markdown.go:95 +0x8b9
github.com/kovetskiy/mark/util.processFile({0xc000529379, 0x7}, 0xc0003f9800, 0xc00009e288, {0x0, 0x0}, {0x0, 0x0}, 0xc0005bf9ef)
        /home/runner/work/mark/mark/util/cli.go:229 +0x19ff
github.com/kovetskiy/mark/util.RunMark({0xc00009e288?, 0x1b6b4c8?}, 0xc00009e288)
        /home/runner/work/mark/mark/util/cli.go:87 +0x6f2
github.com/urfave/cli/v3.(*Command).run(0xc00009e288, {0x1b6b4c8, 0xc000549da0}, {0xc0001181e0, 0x6, 0x6})
        /home/runner/work/pkg/mod/github.com/urfave/cli/v3@v3.3.3/command_run.go:336 +0x2662
github.com/urfave/cli/v3.(*Command).Run(...)
        /home/runner/work/pkg/mod/github.com/urfave/cli/v3@v3.3.3/command_run.go:94
main.main()
        /home/runner/work/mark/mark/main.go:30 +0xe8
```

## Further Information (please complete the following information)

* Mark Version (`mark --version`): 14.0.2
* Mark Parameters: --features d2
* Confluence Hosting: (irrelevant)
* Confluence Version: (irrelevant)
* Environment specific Information: The behavior seems to be identical on Windows and WSL/Debian. Running the d2 unit tests with `go test` DOES work, even though the actual width/height do not match what the test case expects.

--- a/util/cli.go
+++ b/util/cli.go
@@ -222,6 +222,7 @@ func processFile(
 		cfg := types.MarkConfig{
 			MermaidProvider: cmd.String("mermaid-provider"),
 			MermaidScale:    cmd.Float("mermaid-scale"),
+			D2Scale:         cmd.Float("d2-scale"),
 			DropFirstH1:     cmd.Bool("drop-h1"),
 			StripNewlines:   cmd.Bool("strip-linebreaks"),
 			Features:        cmd.StringSlice("features"),
@@ -301,6 +302,7 @@ func processFile(
 	cfg := types.MarkConfig{
 		MermaidProvider: cmd.String("mermaid-provider"),
 		MermaidScale:    cmd.Float("mermaid-scale"),
+		D2Scale:         cmd.Float("d2-scale"),
 		DropFirstH1:     cmd.Bool("drop-h1"),
 		StripNewlines:   cmd.Bool("strip-linebreaks"),
 		Features:        cmd.StringSlice("features"),

